### ISSUE TITLE
Strong heating device in the active.

### ISSUE THREAD

**Issue by Max1337** [CyanogenMod/android_device_samsung_superior#8]
I didn't know how show this , because in log is empty. But due to Kernel Adiutor was observed that average temperature in Celsius is 42-48(In stock 35-38) degrees + battery very fast  discharging "on the eyes". This problem have all your CM's(10.2-12.1)
Please do something !
P.S Sorry for my terrible English.

**Comment by tuxzz** [comment]
屏幕亮起时耗电确实明显。可以试试把cpu频率调节模式换成powersave?
Yeah it really drown the battery fastly while screen is on. You can try change the scaling governors to 'powersave'.

**Comment by tuxzz** [comment]
Yeah it really drown the battery fastly while screen is on. You can try changing the scaling governors to 'powersave'.

**Comment by Max1337** [comment]
I spent my own observations and came to the conclusion. Changing the frequency of the processor and him governors almost nothing has changed. But I found another reason- It's too high battery voltage. On all roms CM include MoKee OS medium voltage is 4.1-4.3 Walt , while on stock medium voltage is 3.8 Walt That is, as stated on the battery itself. Respected Ijzyal , please fix the battery voltage in that this main problem since as CM10.2.
